checker/examples/GradleExamples/GradleJava7Example/build.gradle

```diff
@@ -0,0 +1,60 @@
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+if (JavaVersion.current().isJava8Compatible()) {
+    throw new GradleException("This example requires Java 7, but you are using Java 8. Please see the Java 8 example in ../GradleJava8Example")
+}
+
+assert JavaVersion.current().isJava7()
```

Comment:
When I wrote the separate examples I was assuming two use cases (which I should make explicit):
1. Using JDK 7 on Java 7 source files to produce Java 7 class files and using annotations in comments.
2. Using JDK 8 on Java 8 source files to produce Java 8  class and **not** using annotations in comments.

The Gradle build files could be merged as you suggested, but the source code cannot.   

Would you prefer the Gradle examples demonstrate different use cases?  (Which ever the use cases demonstrated, I'll make it clear in the manual instructions and example code.)